pkg/apis/certmanager/v1/types_certificate.go

```diff
@@ -249,6 +253,37 @@ var (
 	RotationPolicyAlways PrivateKeyRotationPolicy = "Always"
 )
 
+// The type of additional private key format to write
+// Allowed values are `DER` or `PEM`.
+// When Type is set to `DER` an additional entry `key.der` will be created in the secret
+// containing the binary format of the key.
+// When Type is set to `CombinedPEM` and additional entry `tls.pem` will be created in the secret
+// containing the PEM formated certificate (tls.key + tls.crt concatened)
```

Comment:
three typos:

```suggestion
// When Type is set to `CombinedPEM` an additional entry `tls.pem` will be created in the secret
// containing the PEM formatted certificate (tls.key + tls.crt concatenated)
```

Reply:
thanks for spotting all the mistakes :sweat_smile: